src/domain/enrolment/CreateEnrolmentPage.tsx

```diff
@@ -68,18 +69,27 @@ const CreateEnrolmentPage: React.FC = () => {
   const initialValues = React.useMemo(
     () => ({
       ...defaultInitialValues,
+      language: locale.toUpperCase(),
+      // some of the values used only for validation purposes
       minGroupSize: Math.max(
         ...filteredOccurrences.map((item) => item?.minGroupSize || 0)
       ),
+      // figure out maxGroup size based on all occurrences selected
+      // if OccurrenceType is EnrolmentCount, use only maxGroupSize
       maxGroupSize: Math.min(
-        ...filteredOccurrences.map((item) =>
-          Math.min(
-            item?.maxGroupSize || item.amountOfSeats,
-            item.amountOfSeats - (item.seatsTaken || 0)
-          )
-        )
+        ...filteredOccurrences.map((item) => {
+          if (item.seatType === OccurrenceSeatType.ChildrenCount) {
+            return Math.min(
+              item?.maxGroupSize || item.amountOfSeats,
+              item.amountOfSeats - (item.seatsTaken || 0)
+            );
+          }
+          if (item.seatType === OccurrenceSeatType.EnrolmentCount) {
+            return Math.min(item?.maxGroupSize || 0);
```

Comment:
Should this be either  `return Math.min(item?.maxGroupSize, 0);` or just `return item?.maxGroupSize || 0`

Reply:
Wow. I have no idea where this Math.min has come from can't really remember writing myself that because this line doesn't make any sense :D

Reply:
seems like it is some kind of copy paste leftover from the code above, anyway good catch 👍